File: lib/JFFNetwork/Errors/NSNetworkErrors/JNSNoInternetNetworkError.m
#import "JNSNoInternetNetworkError.h"

@implementation JNSNoInternetNetworkError

+ (BOOL)isMineNSNetworkError:(NSError *)error
{
    return [error isNetworkError];
}

- (instancetype)init
{
    return [self initWithDescription:NSLocalizedString(@"JNETWORK_NO_INTERNET_ERROR", nil)];
}

- (void)writeErrorWithJFFLogger
{
    [self writeErrorToNSLog];
}

@end

File: iAsync.Swift/lib/JNetwork/JNetwork/NSError/NSNetworkErrors/JNSNoInternetNetworkError.swift
//
//  JNSNoInternetNetworkError.swift
//  Wishdates
//
//  Created by Vladimir Gorbenko on 18.08.14.
//  Copyright (c) 2014 EmbeddedSources. All rights reserved.
//

import Foundation

class JNSNoInternetNetworkError : JNSNetworkError {
    
    override class func isMineNSNetworkError(error: NSError) -> Bool {
        return error.isNetworkError
    }
    
    override var localizedDescription: String {
        
        return NSLocalizedString(
            "J_NETWORK_NO_INTERNET_ERROR",
            bundle: NSBundle(forClass: self.dynamicType),
            comment:"")
    }
    
    override func writeErrorWithJLogger() {
        writeErrorToNSLog()
    }
}
